docs/ci-integration/guides/gh-actions-integration.md

```diff
@@ -18,11 +18,9 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
-      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
       FORCE_COLOR: 1
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Put back the git branch into git (Earthly uses it for tagging)
```

Comment:
Could we instead do this with the [actions/checkout](https://github.com/actions/checkout#checkout-a-different-branch)?

Reply:
definitely, I updated that in #1856